typing_extensions/setup.py

```diff
@@ -9,23 +9,24 @@
                      'to install the typing package.\n')
     exit(1)
 
-version = '3.7.4.3'
+version = '3.10.0.0'
 description = 'Backported and Experimental Type Hints for Python 3.5+'
 long_description = '''\
 Typing Extensions -- Backported and Experimental Type Hints for Python
 
-The ``typing`` module was added to the standard library in Python 3.5 on
-a provisional basis and will no longer be provisional in Python 3.7. However,
```

Comment:
Off topic: I see https://github.com/python/typing/issues/247 and https://github.com/python/cpython/pull/16204 saying typing became implicitly non-provisional in 3.7 because no new features were added, but PEP 484 still says  ``Status: | Provisional`` ? Maybe that PEP needs discussion/updating once Guido's back.

Reply:
Good catch, I opened python/peps#1942 to fix it.